src/Workleap.Extensions.Mongo.Analyzers.Tests/IndexAttributeUsageAnalyzerTests.cs

```diff
@@ -3,11 +3,13 @@
 public class IndexAttributeUsageAnalyzerTests : BaseAnalyzerTest<IndexAttributeUsageAnalyzer>
 {
     private const string TestClassName = "MyWorker";
+    
+    private const string TestMethodName = "DoSomething";
 
     [Theory]
     [InlineData("[IndexedBy(\"PrimaryKey\")]")]
     [InlineData("[NoIndexNeeded(\"Default index used\")]")]
-    public async Task Given_IndexAttribute_When_Analyze_Then_No_Diagnostics(string attribute)
+    public async Task Given_IndexAttribute_On_Class_When_Analyze_Then_No_Diagnostics(string attribute)
```

Comment:
Test name is wrong, here there is no attribute on the class, there's no {{0}} in the text code. Test works because there is no call being made to a mongo method. 

So change the name and remove theory related stuff here